package.json

```diff
@@ -573,6 +577,11 @@
                     "when": "view == aws.explorer && viewItem == awsStateMachineNode",
                     "group": "1@1"
                 },
+                {
+                    "command": "aws.copyArn",
+                    "when": "view == aws.explorer && viewItem =~ /^(awsRegionFunctionNode|awsCloudFormationFunctionNode|awsStateMachineNode|awsCloudFormationNode|awsCloudFormationNode)$/",
+                    "group": "2@1"
```

Comment:
This may require some adjustments on other items. Clipboard based groups should appear above Delete based groups, but below things like View/Download/Invoke type operations. Can you take a stab at this?

Reply:
I'll see if we can duplicate this entry and change `group` based on the `viewItem`. Are there any other ways to do this?

Visual for reference:
```
View
Download
Invoke
-----------------------
Clipboard
-----------------------
Delete
```

Reply:
Split the commands into three groups based on the suggestion above.

### Lambda Function
![Screen Shot 2020-04-20 at 8 11 26 PM](https://user-images.githubusercontent.com/55506708/79822780-dc962c00-8346-11ea-98bf-27c5b44facb8.png)

### Lambda Function (CloudFormation)
![Screen Shot 2020-04-20 at 8 11 50 PM](https://user-images.githubusercontent.com/55506708/79822785-e029b300-8346-11ea-9196-91531ad9b13f.png)

### CloudFormation Stack
![Screen Shot 2020-04-20 at 8 11 09 PM](https://user-images.githubusercontent.com/55506708/79822776-d99b3b80-8346-11ea-9518-7858122c9572.png)

### Step Functions State Machine
Note: Download Definition and Start Execution were previously separate groups.
![Screen Shot 2020-04-20 at 8 10 49 PM](https://user-images.githubusercontent.com/55506708/79822773-d56f1e00-8346-11ea-8601-8e949e1b409c.png)